Index: src/test/resources/login.feature
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/resources/login.feature	(date 1524865777744)
+++ src/test/resources/login.feature	(date 1524865777744)
@@ -0,0 +1,17 @@
+Feature: Login to account
+
+  Scenario Outline: Open tabs to login to trade coin club and email
+    Given I navigate to site with user <username>
+    When I enter username
+    And I enter password
+    And I press sign in
+    And I wait for google authenticator to be completed
+    Then I open a new tab
+    And I navigate to email login portal
+    And I enter email username
+    And I enter email password
+    And I confirm I have logged in
+
+    Examples:
+    |username|
+    |missatwo|
\ No newline at end of file
Index: src/test/java/definition/LoginStepDefinitions.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/definition/LoginStepDefinitions.java	(date 1524866389180)
+++ src/test/java/definition/LoginStepDefinitions.java	(date 1524866389180)
@@ -0,0 +1,37 @@
+package definition;
+
+import cucumber.api.PendingException;
+import cucumber.api.java.en.Then;
+
+public class LoginStepDefinitions extends DefinitionUtils{
+
+    @Then("^I open a new tab$")
+    public void i_open_a_new_tab() throws Throwable {
+        // Write code here that turns the phrase above into concrete actions
+        throw new PendingException();
+    }
+
+    @Then("^I navigate to email login portal$")
+    public void i_navigate_to_email_login_portal() throws Throwable {
+        // Write code here that turns the phrase above into concrete actions
+        throw new PendingException();
+    }
+
+    @Then("^I enter email username$")
+    public void i_enter_email_username() throws Throwable {
+        // Write code here that turns the phrase above into concrete actions
+        throw new PendingException();
+    }
+
+    @Then("^I enter email password$")
+    public void i_enter_email_password() throws Throwable {
+        // Write code here that turns the phrase above into concrete actions
+        throw new PendingException();
+    }
+
+    @Then("^I confirm I have logged in$")
+    public void i_confirm_I_have_logged_in() throws Throwable {
+        // Write code here that turns the phrase above into concrete actions
+        throw new PendingException();
+    }
+}
